apps/evans/components/core/EmbeddedAudio.tsx

```diff
@@ -1,60 +1,176 @@
+            >
+              <i
+                className={cx(
+                  `fa-solid text-lg text-white`,
+                  isPlaying ? `fa-pause` : `fa-play`,
+                )}
+              />
+            </button>
+            <div className="flex-grow bg-flprimary/5 py-2 px-6 rounded-full flex justify-start overflow-hidden relative mx-8 hidden md:block md:order-1">
+              <span className="text-xl font-bold text-flprimary whitespace-nowrap">
+                {currentTitle}
+              </span>
+              <div className="absolute h-full w-24 right-0 top-0 bg-gradient-to-l from-[#F7F5F5] to-transparent" />
+            </div>
+            <button
+              onClick={() => {
+                setCurrentTime(currentTime - 10);
+                audio.currentTime = currentTime - 10;
+              }}
+              className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center md:mr-2 order-0 md:order-2 shrink-0"
+            >
+              <BackwardIcon className="w-6 h-6 text-flprimary" />
+            </button>
+            <button
+              onClick={() => {
+                setCurrentTime(currentTime + 10);
+                audio.currentTime = currentTime + 10;
+              }}
+              className="w-10 h-10 rounded-full hover:bg-flprimary/10 active:bg-flprimary/20 active:scale-95 transition-[background-color,transform] flex justify-center items-center order-2 md:order-3 shrink-0"
+            >
+              <ForwardIcon className="w-6 h-6 text-flprimary" />
+            </button>
+          </div>
+          <div className="flex-grow flex items-center">
+            <span className="text-flgray-500 w-14 flex justify-start">
+              {formatDuration(currentTime)}
+            </span>
+            <div className="flex-grow flex items-center relative">
+              <div
+                className="absolute left-0 top-[-1px] rounded-l-full bg-flprimary pointer-events-none h-[8px]"
+                style={{
+                  width: `${(currentTime / audio.duration) * 100}%`,
+                }}
+              />
+              <input
+                type="range"
+                className={cx(`flex-grow`, styles.progressBar)}
+                min={0}
+                max={audio.duration}
+                value={currentTime}
+                onChange={(e) => {
+                  audio.currentTime = Number(e.target.value);
+                }}
+              />
+            </div>
+            <span className="text-flgray-500 w-14 flex justify-end">
+              {formatDuration(audio.duration)}
+            </span>
+          </div>
+        </div>
+      ) : (
+        <div>loading</div>
+      )}
+      {tracks.length > 1 &&
```

Comment:
can you make a storybook story for the case where there is only one track? that's the most common case, and i can't currently see what it looks like, since we're not pulling real data.